### PR TITLE
Add components for the main page

### PR DESCRIPTION
**Pull request by TanyaKalistruk**
In this PR components for main page are added with some defualt params, without any integration with API

### CODE REVIEW

File: MyBooking/Views/BookingView/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (KovalenkoAndrii) [inline MyBooking/Views/BookingView/.gitignore]: please do not use two files

File: MyBooking/Views/BookingView/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+In the 'Views/BookingView' directory, you can run:
+
+### `npm install`
+
+Installs dependensis for the UI part of project
+
+### `npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```
Comment (KovalenkoAndrii) [inline MyBooking/Views/BookingView/README.md:12]: our api runs on 49561 or am i wrong?

File: MyBooking/Views/BookingView/src/App.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+                price: 100,
+                peopleCount: 1,
+                possibleAnimals: true,
+                key: 'asdc123',
```
Comment (KovalenkoAndrii) [inline MyBooking/Views/BookingView/src/App.js:43]: Is this field somehow related to our tables?

Comment (KovalenkoAndrii) [reply MyBooking/Views/BookingView/src/App.js:43]: `key: 'asdc123`

File: MyBooking/Views/BookingView/.gitignore (hunk 1 above)
Comment (TanyaKalistruk) [reply MyBooking/Views/BookingView/.gitignore]: Okay, I will delete it

File: MyBooking/Views/BookingView/README.md (hunk 2 above)
Comment (TanyaKalistruk) [reply MyBooking/Views/BookingView/README.md:12]: You are right, but I think it would be better to run the app on the one port and call to another, it would be easier not to confuse and clearly understand that we can get our data in the one place

File: MyBooking/Views/BookingView/src/App.js (hunk 3 above)
Comment (TanyaKalistruk) [reply MyBooking/Views/BookingView/src/App.js:43]: It is a temporary value when we make integration with API it will be changed to some id. It's needed for rendering unique cells in react

Comment (TanyaKalistruk) [reply MyBooking/Views/BookingView/src/App.js:43]: And no, it is not related to out tables

File: MyBooking/Views/BookingView/.gitignore (hunk 1 above)
Comment (TanyaKalistruk) [reply MyBooking/Views/BookingView/.gitignore]: Done

**Review by AndriiUngurian — APPROVED** [review]

**Review by KovalenkoAndrii — APPROVED** [review]
